spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
@syxanash 
Thanks, but it's not enough.
Please make a new Pull Request by [your branch](https://github.com/syxanash/itunes-client/tree/volume-controls).